ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -281,70 +306,66 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-edge-zone-byo-subnet-ovn-day2
-- as: aws-upi-basecap-none-amd-f28
-  cron: 24 15 12 * *
+- as: aws-upi-basecap-none-arm-f28
+  cron: 2 1 15 * *
   steps:
     cluster_profile: aws-qe
     env:
-      ADDITIONAL_ENABLED_CAPABILITIES: CloudCredential
+      ADDITIONAL_ENABLED_CAPABILITIES: CloudCredential CloudControllerManager Ingress
       BASE_DOMAIN: qe.devcluster.openshift.com
       BASELINE_CAPABILITY_SET: None
+      BOOTSTRAP_INSTANCE_TYPE: m6g.large
+      MASTER_INSTANCE_TYPE: m6g.xlarge
+      OCP_ARCH: arm64
       PERSISTENT_MONITORING: "false"
+      WORKER_INSTANCE_TYPE: m6g.xlarge
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-upi
-- as: azure-ipi-arm-mixarch-disc-fullypriv-f28-day2-64k-pagesize
-  cron: 4 11 5 * *
+- as: azure-ipi-basecap-vset-additionalcap-arm-f28
+  cron: 26 12 15 * *
   steps:
     cluster_profile: azure-qe
     env:
-      ADDITIONAL_WORKER_ARCHITECTURE: x86_64
-      ADDITIONAL_WORKER_VM_TYPE: Standard_D4s_v3
-      ADDITIONAL_WORKERS: "1"
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
       COMPUTE_NODE_TYPE: Standard_D4ps_v5
-      MCO_CONF_DAY2_CUSTOM_MCP: '[{"mcp_name": "64k-pages", "mcp_node_num": "1", "mcp_node_label":
-        "node-role.kubernetes.io/worker=,kubernetes.io/arch=arm64"}]'
-      MCO_CONF_DAY2_INSTALL_KERNEL_MCPS: 64k-pages
-      MCO_CONF_DAY2_INSTALL_KERNEL_TYPE: 64k-pages
       OCP_ARCH: arm64
-      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
+      PERSISTENT_MONITORING: "false"
     test:
-    - chain: ipi-install-heterogeneous-day2-kerneltype
-    - chain: openshift-upgrade-qe-test-disconnected
-    workflow: cucushift-installer-rehearse-azure-ipi-disconnected-fullyprivate
-- as: azure-ipi-confidential-trustedlaunch-f28
-  cron: 29 2 24 * *
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-azure-ipi-capability-baselinecaps-vset-additionalcaps
+- as: azure-ipi-basecap-vset-arm-f28
+  cron: 46 23 7 * *
   steps:
     cluster_profile: azure-qe
     env:
+      ADDITIONAL_ENABLED_CAPABILITIES: CloudCredential CloudControllerManager Ingress
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: Standard_D4ps_v5
+      OCP_ARCH: arm64
+      PERSISTENT_MONITORING: "false"
     test:
     - chain: openshift-upgrade-qe-test
-    workflow: cucushift-installer-rehearse-azure-ipi-confidential-trustedlaunch
-- as: azure-ipi-des-mini-perm-fips-amd-f28
-  cron: 26 14 2 * *
+    workflow: cucushift-installer-rehearse-azure-ipi-capability-baselinecaps-vset
+- as: azure-ipi-confidential-trustedlaunch-f28
```

Comment:
This one is duplicated with the one in amd upgrade ci, should be dropped.

Reply:
Could you let me know which job is duplicated with?

Reply:
Could you let me know which job is duplicated with?

Reply:
https://github.com/openshift/release/pull/54740/files#diff-263e51ea063c56c1c13193a1577b833d5248aefde68254d778355a6171caabd3R358